app/modules/parsing/graph_construction/parsing_controller.py

```
@@ -237,6 +238,51 @@ def _on_email_done(t: asyncio.Task) -> None:
                         "Returning existing project."
                     )
+                    # Ensure worktree exists in repo manager when enabled
+                    if os.getenv("REPO_MANAGER_ENABLED", "false").lower() == "true":
+                        repo_name = str(project.repo_name) if project.repo_name is not None else None
+                        branch = str(project.branch_name) if project.branch_name is not None else None
+                        commit_id_val = str(project.commit_id) if project.commit_id is not None else None
+                        repo_path = str(project.repo_path) if project.repo_path is not None else None
+                        if repo_name and not repo_path:
+                            ref = commit_id_val if commit_id_val else branch
+                            if ref:
+                                from app.modules.code_provider.github.github_service import GithubService  # noqa: PLC0415
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)